btree/btree.c

```diff
@@ -3470,7 +3475,7 @@ static int64_t btree_put_kv_tick(struct m0_sm_op *smop)
 			node_slot.s_node = oi->i_nop.no_node;
 
 			node_lock(lev->l_node);
-			lev->l_seq = lev->l_node->n_seq;
+			lev->l_seq = oi->i_nop.no_node->n_seq;
```

Comment:
Why are we doing this change?
lev->l_node is not getting modified.

Reply:
Yes. lev->l_node is same as oi->i_nop.no_node but as at line no, 3474-3475, we are initializing  contents of lev as per oi->i_nop.no_node, so to provide more readability this change is made.

Reply:
Ok